fatiando/tests/mesher/test_oblate_ellipsoid.py

```diff
@@ -0,0 +1,49 @@
+from ...mesher import OblateEllipsoid
```

Comment:
You can put all tests in the same file `test_ellipsoids.py` because they are very similar. That way it's easier to maintain.